playbooks/provision_openstack_instance.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: "Provision Open Stack Instance"
+  hosts: '{{ hosts | default("all") }}'
+  become: '{{ become | default("yes") }}'
+  gather_facts: '{{ gather_facts | default("no") }}'
+  tasks:
+
+    - name: "provision a key pair"
+      os_keypair:
+        state: present
+        name: ansible_key
+      register: keypair_output
+
+    - name: "copy the keypair_output to a file"
+      copy:
+        content: keypair_output["private_key"]
+        dest: "~/.ssh/keypair_output.pem"
+
+    - name: "Create a new OpenShift instance on Virtual Machine"
+      os_server:
+        state: present
+        name: "{{ vm_name }}"
+        image: minishift-CentOS-7
+        key_name: ansible_key
+        flavor: ci.m1.xlarge
+        network: provider_net_cci_2
+        security_groups: minishift
```

Comment:
The `image`, `flavor`, `network` and `security_groups` are all specific to the OpenStack cluster instance the team is currently using. I'd suggest making variables for these (like for the `name` key on line 22), and setting the above values as defaults.